### ISSUE TITLE
Update volume/chapter progress from "Watched Series" page

### ISSUE THREAD

**Issue by smmalmansoori** [fake-name/wlnupdates#6]
Maybe something similar to how it's done in MangaUpdates "Reading List"?

**Comment by fake-name** [comment]
v/c?

**Comment by smmalmansoori** [comment]
Volume and/or Chapter. I will update the title.

**Comment by fake-name** [comment]
oh, durrrr

I'll poke something like this in when I get time (probably this weekend).

**Comment by smmalmansoori** [comment]
Thanks!

I'm a Flask developer too, I hope I'll try to contribute when I get some time.

**Comment by fake-name** [comment]
That'd be badass!

**Comment by fake-name** [comment]
I haven't had time to look at this yet, sorry.

I moved the backend feed system to use a distributed RPC fetching system, and I've subsequently been tracking down a pile of really annoying issues relating to strange socket behaviour in the RPC message passing libraries - after being open for a number of hours, it would silently die without an error message, which would lead to all the RSS updates halting.

Anyways, I think most of the issues have been fixed at this point, but with the updater dying every few hours fixing it was somewhat critical.
